slim/train_image_classifier.py

```diff
@@ -457,12 +457,21 @@ def clone_fn(batch_queue):
       # Specify the loss function #
       #############################
       if 'AuxLogits' in end_points:
-        tf.losses.softmax_cross_entropy(
-            logits=end_points['AuxLogits'], onehot_labels=labels,
-            label_smoothing=FLAGS.label_smoothing, weights=0.4, scope='aux_loss')
+          logits = end_points['AuxLogits']
```

Comment:
The logits must be changed if the condition is True because we have to feed the new logits to calculated accuracy as it is needed to be fed for calculation of softmax cross-entropy loss.